feature/auth/src/main/java/com/teamoffroad/feature/auth/presentation/SetBirthDateViewModel.kt

```diff
@@ -136,4 +136,20 @@ class SetBirthDateViewModel @Inject constructor(
             }
         }
     }
+
+    fun initState() {
+        if (birthDateUiState.value.yearValidateResult != DateValidateResult.Success &&
+            birthDateUiState.value.monthValidateResult != DateValidateResult.Success &&
+            birthDateUiState.value.dayValidateResult != DateValidateResult.Success
+        )
+            _birthDateUiState.value = birthDateUiState.value.copy(
```

Comment:
```suggestion
            _birthDateUiState.value = SetBirthDateUiState()
```
로 하시고 기본값을 데이터클래스에 넣어주시는건 어떤가요!?

Reply:
오.. 그방법이